PhysicsTools/NanoAOD/python/nanoDQM_cfi.py

```diff
@@ -240,6 +240,17 @@
                 Plot1D('tau4', 'tau4', 20, 0, 1, 'Nsubjettiness (4 axis)'),
             )
         ),
+        PFCand = cms.PSet(
+            sels = cms.PSet(),
+            plots = cms.VPSet(
+                Count1D('_size', 10, 0, 100, 'PF candidates'),
+                Plot1D('pt', 'pt', 20,  0, 50, 'Puppi-weighted pt'),
+                Plot1D('eta', 'eta', 20, -4,  4., 'eta'),
+                Plot1D('phi', 'phi', 20, -3.14159, 3.14159, 'phi'),
+                Plot1D('mass', 'mass', 10, 0, 1., 'Puppi-weighted mass'),
+                Plot1D('pdgId', 'pdgId', 44, -220, 220, 'PF candidate type (+/-211 = ChgHad, 130 = NeuHad, 22 = Photon, +/-11 = Electron, +/-13 = Muon, 1 = HFHad, 2 = HFEM)'),
+            )
+        ),
```

Comment:
@nurfikri89 Would it make sense to add `FatJetPFCand_PFCandIdx` and `FatJetPFCand_jetIdx` to DQM as well?

Reply:
Looking at the indices that we store for other objects, we do not make DQM plots for them. I do not see a reason to do anything differently here.